descriptors/ModuleDescriptor-template.json

```diff
@@ -1,6 +1,28 @@
 {
   "id": "${artifactId}-${version}",
   "name": "Circulation Storage Module",
+  "requires": [
+    {
+      "id": "login",
+      "version": "7.0"
+    },
```

Comment:
Could this dependency be removed now, since we don't log in anymore?

Reply:
Yes, sure